README.md

```diff
@@ -151,33 +151,6 @@ repo, open an issue within the Azure CLI repo.
 - [Azure PowerShell issues](https://github.com/Azure/azure-powershell/issues)
 - [Azure CLI issues](https://github.com/Azure/azure-cli/issues)
 
-### Types of tools
-
-Cloud Shell aims to provide a core set of tools for Azure and Microsoft 365 devops scenarios, but we
-can't include everything. If you just want to use a tool yourself, you can install most utilities
-into your own home directory inside Cloud Shell. You only need to update the image if you want
-_every_ Cloud Shell admin to have the tool available.
-
-For a tool to be included in Cloud Shell, it has to be:

```

Comment:
This seems to duplicate the content in docs/package-inclusion-guide.md, in shorter form. Should probably simply reference that file rather than duplicate and drift.

Reply:
Hey Ross, this is what we had before and is now deleted.